.NET/Microsoft.Recognizers.Text.DateTime/Parsers/BaseTimeZoneParser.cs

```diff
@@ -14,7 +14,7 @@ public class BaseTimeZoneParser : IDateTimeParser
         public static readonly Regex TimeZoneEndRegex = new Regex("time$|timezone$", RegexOptions.Singleline);
 
         // Compute UTC offset in minutes from matched timezone offset in text. e.g. "-4:30" -> -270; "+8"-> 480.
-        public static int ComputeMinutes(string utcOffset)
+        public static int ComputeMinutes(string timeZone, string utcOffset)
```

Comment:
This method has a very clear intent to do simple calculations. I don't think passing timezone names into it is a good design.